scheduler/bin/run-docker.sh

```diff
@@ -33,6 +33,7 @@ docker run \
     --publish=${COOK_PORT}:${COOK_PORT} \
     -e "COOK_PORT=${COOK_PORT}" \
     -e "COOK_NREPL_PORT=${COOK_NREPL_PORT}" \
+    -e "COOK_FRAMEWORK_ID=cook-framework-${COOK_PORT}" \
```

Comment:
I don't follow why we should ever want to set our own `framework-id` instead of letting Mesos set it for us.